src/control-plane/backend/lambda/api/store/aws/s3.ts

```diff
@@ -105,3 +106,21 @@ export const getS3BucketPolicy = async (region: string, bucket: string) => {
     return undefined;
   }
 };
+
+export const isBucketExist = async (region: string, bucket: string) => {
+  try {
+    const s3Client = new S3Client({
+      ...aws_sdk_client_common_config,
+      region,
+    });
+    const params: GetBucketLocationCommand = new GetBucketLocationCommand({
+      Bucket: bucket,
+      ExpectedBucketOwner: awsAccountId,
+    });
+    await s3Client.send(params);
+    return true;
+  } catch (error) {
+    logger.error('get S3 bucket location error ', { error });
```

Comment:
Let's change it to `warn`. It's not an internal error in our code; it's a known use case.